internal/pkg/plugin/plugin_test.go

```diff
@@ -101,3 +102,29 @@ func TestSearchPath(t *testing.T) {
 	require.True(t, ok)
 	require.Equal(t, fileName, filepath.Base(pluginPaths[0]))
 }
+
+func TestVersionRegex(t *testing.T) {
```

Comment:
FYI, these tests don't add any coverage. It might be better to write a helper function and unit test that!